### PR TITLE
Create rubbish

### PR DESCRIPTION
**Pull request by aleshakay**
Added createRubbish 

** using POST 
** https://localhost:44300/api/rubbish/
** body similar too ```{
    "rubbishId": 11,
    "name": "Out Miamn Shirt",
    "description": "This sleeveless; with oceanon it.",
    "categoryId": 2,
    "isAvailable": true,
    "userId": 2,
    "price": 46
}```

also 

https://localhost:44300/api/User/updateUser

### CODE REVIEW

File: Bangazon-Tinkr/Controllers/RubbishController.cs [hunk 1]
```diff
@@ -41,6 +45,21 @@ public IActionResult GetAllRubbish()
 
             return Ok(allRubbish);
         }
+        // api/Rubbish/2/createRubbish
+        [HttpPost("{userId}/createRubbish")]
```
Comment (broach44) [inline Bangazon-Tinkr/Controllers/RubbishController.cs]: - I think the addition of the userId and the /createRubbish are not necessary for the route.  You are already completing a Post request on the Rubbish controller so the route is different than the posts for through the other controllers.

File: Bangazon-Tinkr/Controllers/RubbishController.cs [hunk 2]
```diff
@@ -41,6 +45,21 @@ public IActionResult GetAllRubbish()
...
+        public IActionResult CreateNewRubbish(int userId, Rubbish rubbishToAdd)
```
Comment (broach44) [inline Bangazon-Tinkr/Controllers/RubbishController.cs]: If we remove from the route we can remove the userId from here also.  You would want to have the userId in the body of what is being sent to create the new piece of rubbish.

File: Bangazon-Tinkr/Controllers/RubbishController.cs [hunk 3]
```diff
@@ -41,6 +45,21 @@ public IActionResult GetAllRubbish()
...
+        {
+            var user =  _usersRepository.GetUserById(userId);
```
Comment (broach44) [inline Bangazon-Tinkr/Controllers/RubbishController.cs]: If the above changes are made you can use something like rubbishToAdd.userId to check for valid user.

**Comment by jthielman** [comment]
👍

**Pull request merged**